### ISSUE TITLE
Sofc Keras Surrogates failing tests

### ISSUE THREAD

**Issue by rundxdi** [IDAES/idaes-pse#1431]
Tests on sofc_keras_surrogate.py are failing due to an out-of-date Keras model that needs to be re-generated by someone who knows about the model.  A deprecation warning indicating these failures has been added.

**Comment by ksbeattie** [comment]
Discovered in #1401

**Comment by ksbeattie** [comment]
This will need to wait for #1401 to be merged before re-generating the model (and other keras related ones).

**Comment by lbianchi-lbl** [comment]
This should be done independently of #1401 so that #1401 is not held up. A separate PR after #1401 is merged should be created once @AlexNoring recreates the model using the updated (post-#1401) supported version of Keras/Tensorflow.

**Comment by AlexNoring** [comment]
The SOFC flowsheets in the examples repository no longer use the keras surrogates. I'd suggest we simply remove the files.

**Comment by ksbeattie** [comment]
@bpaul4 will open a PR addressing this.
